tests/test_csc.py

```diff
@@ -34,28 +35,36 @@
 
 class CscTestCase(salobj.BaseCscTestCase, unittest.IsolatedAsyncioTestCase):
     def basic_make_csc(
-        self, initial_state, config_dir, simulation_mode, settings_to_apply="", **kwargs
+        self,
+        initial_state,
+        config_dir,
+        simulation_mode,
+        index=1,
```

Comment:
Please consider an enum value for the default (or does that make mypy unhappy? If so...another salobj update).

and also enum values for the other times you set the index.